site/en/docs/extensions/reference/action/index.md

```diff
@@ -2,18 +2,16 @@
 api: action
 ---
 
-You can use the `chrome.action` API to control the toolbar button for your extension in Chrome's UI.
-The action icons are displayed in the browser toolbar, to the right of the omnibox (on left-to-right
-devices). After installation, by default, these appear in the extensions menu (the puzzle piece).
-Users can choose to pin your extension icon to the toolbar.
+To control your extension's toolbar button in Chrome's UI, use the `chrome.action` API.
+The action icons are displayed in the browser toolbar next to the omnibox. <!--can I just say "address bar," or is there a reason to use the technical term?--> After installation, by default, these appear in the extensions menu (the puzzle piece icon).
+Users can pin your extension icon to the toolbar.
 
-Note that every extension will have an icon in the toolbar in Chrome, even if the `action` key is
-not specified.
+Every extension has an icon in the Chrome toolbar, even if the `action` key isn't specified.
```

Comment:
The style guide discourages "above" and "below" referring to other parts of the same document, because they cause problems with screen readers. They're also not very specific: what "below" are we telling the reader to look at? I don't think you mean the following section, because that doesn't have an example without an action key.